nixos/modules/services/wayland/cage.nix

```
@@ -0,0 +1,82 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cage;
+in {
+  options.services.cage.enable = mkEnableOption ''
```

`mkEnableOption` already prefixes "Enable " to the description - it can probably just be

```suggestion
  options.services.cage.enable = mkEnableOption '"Cage Kiosk";
```

---

Adressed in  #81327.